### PR TITLE
Gas price threshold for transactions

### PR DESCRIPTION
**Pull request by tomusdrw**
1. Threshold specified using CLI
2. Dropping transaction from queue when it's gas_price is lower than specified threshold.

When #637 is merged new TransactionQueue specific error type should be introduced.

### CODE REVIEW

File: parity/main.rs [hunk 1]
```diff
@@ -108,6 +108,7 @@ API and Console Options:
   --rpccorsdomain URL      Equivalent to --jsonrpc-cors URL (geth-compatible).
 
 Sealing/Mining Options:
+  --gasprice GAS		   Minimal gas price a transaction must have to be accepted for mining [default: 50000000000].
```
Comment (gavofyork) [inline parity/main.rs]: is this in keeping with geth?

File: parity/main.rs [hunk 2]
```diff
@@ -248,6 +250,11 @@ impl Configuration {
 		Address::from_str(&self.args.flag_author).unwrap_or_else(|_| die!("{}: Invalid address for --author. Must be 40 hex characters, without the 0x at the beginning.", self.args.flag_author))
 	}
 
+	fn gasprice(&self) -> U256 {
+		U256::from_dec_str(self.args.flag_gasprice).unwrap_or_else(|_| die("{}: Invalid gasprice given. Must be a
```
Comment (gavofyork) [inline parity/main.rs]: two lines?

File: parity/main.rs (hunk 1 above)
Comment (tomusdrw) [inline parity/main.rs]: Yes. Taken from geth@1.3.3. Will check on `1.4` in the morning.

File: parity/main.rs (hunk 2 above)
Comment (tomusdrw) [inline parity/main.rs]: vim + .editorconfig. Will break the line in different place.

File: parity/main.rs (hunk 1 above)
Comment (tomusdrw) [inline parity/main.rs]: Updated to 20kkk (geth@1.3.5)

**Comment by tomusdrw** [comment]
Note it should be merged before #653

**Pull request merged**